docs/src/manual.md

```diff
@@ -147,13 +147,21 @@ Note that some of MOI constraints can be bridged, see [Bridges](http://www.julia
 |   `SingleVariable`   |
 |   `ScalarAffineFunction`   |
 
-## Dualize a MathOptInterface model
-
-Conic dual of a model
-ADD EXAMPLE.
+## Dualize a model
 
+```@docs
+dualize
+```
 
 ## DualOptimizer
 
-You can solve primal problems using its dual formulation.
+You can solve primal problems using its dual formulation. 
 ADD EXAMPLE.
+
+## Adding new sets
+
+As Dualization.jl it build entirely on top of MathOptInterface.jl automatically dualize models with custom sets. 
```

Comment:
it -> is
build -> built

Reply:
this sentence doesn't make sense even with @blegat's suggestions